### PR TITLE
fix(surveys): survey bug fixes

### PR DESCRIPTION
**Pull request by liyiy**
## Problem

bug fix cleanup

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/scenes/surveys/surveyLogic.tsx [hunk 1]
```diff
@@ -165,11 +138,12 @@ export const surveyLogic = kea<surveyLogicType>([
         updateSurveySuccess: async ({ survey }) => {
             lemonToast.success(<>Survey {survey.name} updated</>)
             actions.editingSurvey(false)
-            router.actions.replace(urls.survey(survey.id))
```
Comment (EDsCODE) [inline frontend/src/scenes/surveys/surveyLogic.tsx:168]: Not familiar with issue here but is the route not necessary anymore?

File: frontend/src/scenes/surveys/surveyLogic.tsx [hunk 2]
```diff
@@ -56,6 +56,29 @@ export const getSurveyEventName = (surveyName: string): string => {
...
+            after: '-30d',
+            limit: 100,
+            event: getSurveyEventName(surveyName),
+        },
+        propertiesViaUrl: true,
+        showExport: true,
+        showReload: true,
+        showColumnConfigurator: true,
+        showEventFilter: true,
+        showPropertyFilter: true,
```
Comment (EDsCODE) [inline frontend/src/scenes/surveys/surveyLogic.tsx:77]: nit: i think you can use the "full" param to show almost all of these

**Review by EDsCODE — APPROVED** [review]

File: frontend/src/scenes/surveys/surveyLogic.tsx (hunk 1 above)
Comment (liyiy) [reply frontend/src/scenes/surveys/surveyLogic.tsx:168]: I noticed it doesn't seem to do anything 🤔

File: frontend/src/scenes/surveys/surveyLogic.tsx [hunk 3]
```diff
@@ -56,6 +56,29 @@ export const getSurveyEventName = (surveyName: string): string => {
     return `${surveyName} survey sent`
 }
 
+const SURVEY_ANSWER_PROPERTY = '$survey_answer'
```
Comment (neilkakkar) [inline frontend/src/scenes/surveys/surveyLogic.tsx:59]: bit confused what's this for, the site app & docs say the response prop is `$survey_response` ?

**Pull request merged**